packages/sentry/index.tsx

```diff
@@ -2,6 +2,8 @@ import * as Sentry from '@sentry/nextjs';
 import { urlFromContext } from '@ifixit/helpers/nextjs';
 import { GetServerSidePropsContext } from 'next';
 import { Scope } from '@sentry/nextjs';
+import { SentryErrorIntegration as Integration } from './SentryErrorIntegration';
```

Comment:
why do we import and export this as `Integration`?

Reply:
I probably should have used export *

Then main reason was that I am skeptical that devs know what is exported. Both eslint and TS do know but at times its unapparent.